### ISSUE TITLE
TestQgsGdalProvider::testGdalProviderQuerySublayers() fails [ netcdf ]

### ISSUE THREAD

**Issue by strk** [qgis/QGIS#48905]
### What is the bug or the crash?

```
FAIL!  : TestQgsGdalProvider::testGdalProviderQuerySublayers() Compared values are not the same
   Actual   (res.count()): 0
   Expected (8)          : 8
   Loc: [/usr/local/src/qgis/qgis/src/master/tests/src/core/testqgsgdalprovider.cpp(486)]
```

### Steps to reproduce the issue

Run: `xvfb-run ctest --output-on-failure -R test_core_gdalprovider`

### Versions

Current master branch as of Jum 7th 2022 - e58cb1f02d00cc7256a4e339228cc7dcb0ccd683

### Supported QGIS version

- [X] I'm running a supported QGIS version according to the roadmap.

### New profile

- [ ] I tried with a new QGIS profile

### Additional context

gdal-config --version returns 3.2.2

**Comment by strk** [comment]
This problem is fixed with the change proposed in #48907 (skip netcdf based tests when driver is unavailable)

**Comment by strk** [comment]
PR #48907 was merged, this issue is fixed (sorry got the wrong issue id in commit)
